src/test_l1_dsVideoPort.c

```diff
@@ -315,42 +329,42 @@ void test_l1_dsVideoPort_negative_dsVideoPortTerm(void) {
  * 
  */
 void test_l1_dsVideoPort_positive_dsGetVideoPort(void) {
-	gTestID = 5;
-	UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-
-	dsError_t status;
-
-	intptr_t handle[NUM_OF_PORTS]={HANDLE_ARRAY_INIT};
-	intptr_t lastHandle;
-	intptr_t newHandle;
-
-	// Step 01: Initialize video port system
-	status = dsVideoPortInit();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	// Step 02: Get the video port handle for valid video port type and index
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsGetVideoPort(kPorts[i].id.type, kPorts[i].id.index, &(handle[i]));
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-		if(i == NUM_OF_PORTS-1)
-		{
-			lastHandle = handle[i];
-		}
-	}
-
-	// Step 03: Compare with the last handle
-	status = dsGetVideoPort(kPorts[NUM_OF_PORTS-1].id.type, kPorts[NUM_OF_PORTS-1].id.index, &(handle[NUM_OF_PORTS-1]));
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	//getting last handle in to new handle for comparision
-	newHandle = handle[NUM_OF_PORTS-1];
-	UT_ASSERT_EQUAL(lastHandle , newHandle);
-
-	// Step 04: Terminate the video port system
-	status = dsVideoPortTerm();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	UT_LOG("\n Out %s\n", __FUNCTION__); 
+    gTestID = 5;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status;
+
+    intptr_t handle[gDSvideoPort_NumberOfPorts];
+    intptr_t lastHandle;
+    intptr_t newHandle;
+
+    // Step 01: Initialize video port system
+    status = dsVideoPortInit();
+    UT_ASSERT_EQUAL(status, dsERR_NONE);
```

Comment:
use  UT_ASSERT_EQUAL_FATAL() for dsVideoPortInit(), dsVideoPortTerm() 
same way like L2 testcases

Reply:
done